### ISSUE TITLE
[ENHANCEMENT] Disable or remove resource header

### ISSUE THREAD

**Issue by beau-gosse** [modxcms/revolution#12660]
### Summary
The MODX resource header -- the area with the rendered title of the resource being edited -- isn't very useful in its current state when the resource has a very long `pagetitle`. I realize that the `longtitle` field was designed to take care of long titles, but with Blog setups, `longtitle` is rarely used.

### Step to reproduce
Select a resource from the tree (or right click it, then choose "Edit") to enter edit mode.
 
### Observed behavior
As the following image shows, the toolbar buttons hide a part of the rendered resource title when the pagetitle has more than 40 characters for the average screen resolution.

![the-need-to-remove-modx-resource-header](https://cloud.githubusercontent.com/assets/509451/9988682/b0dea8d8-6020-11e5-9c81-883df06905a1.png)

In the age of blogging, I don't see much use for the rendered title section (at least not in its current configuration). I propose disabling it by default via a yes/no system setting e.g. `show_resource_header`. We also gain some screen realestate with the header gone. 

Another possible solution would be grouping toolbar buttons in a split/dropdown unified button as shown in the following Fiddle concept: https://jsfiddle.net/sxfznkq5/

**Comment by whitebyte** [comment]
I like the way ~~Wordpre~~ The Unnameable CMS handles this: 
![wp](https://cloud.githubusercontent.com/assets/7704186/9992474/e44aa594-608b-11e5-8fac-d1b421a5fa90.jpg)

The `pagetitle` field only shown as an editable input, but this input is large and noticeable.

**Comment by whitebyte** [comment]
> I suggest controlling it via a system settings

Please don't. System setting are already bloated and overused.

**Comment by sottwell** [comment]
I don't see any reason for this. If your pagetitle is too long, then use the longtitle. If your "blogging solution" can't use the longtitle then maybe there's something wrong with the blogging solution. It's not any great leap to convert [[+pagetitle]] to [[+longtitle]].

**Comment by zaigham** [comment]
IMO there is duplication of information here and end user might confuse that the input in editable field is not being reflected in rendered title. So we can take this UX enhancement for 2.5.x probably.

Regarding the buttons, I think if we remove the rendered title, the tab headers will move up and will get behind the buttons. I would invite designers who use MODX to share some ideas to improve this part of the resource editing.

**Comment by beau-gosse** [comment]
Some valid points were made here. Perhaps I shouldn't have turned this into a `longtitle` vs `pagetitle` match. I do agree that the rendered title has its benefits when used properly. I still believe that something can be done along the lines of achieving a more streamlined UI. I see great potential in using a split button to group toolbar buttons. We can gain lots of room from this. I guess my initial post was questioning the effectiveness and usefulness of the rendered title (as currently configured) when part of it isn't visible. WordPress does indeed have a clever take on this -- maybe we can take some notes.

**Comment by OptimusCrime** [comment]
I'd like to get rid of `Help` and `Close` too.

**Comment by pixelchutes** [comment]
Isn't <kbd>Close</kbd> used to help maintaining "locked" state when working with teams? (e.g. `action=resource/locks/release`) ...Otherwise, there's not a clean way to ensure a lock is released if a user simply closes their open tab(s).

**Comment by whitebyte** [comment]
AFAIK when you leave a resource (either with `close` button or just by opening another link) it becomes unlocked, and `close` just returns us to the main page.

**Comment by whitebyte** [comment]
This also conflicts with #12646

**Comment by jpdevries** [comment]
> BTW I personally find close and help buttons utterly useless.

Agreed. I'd +1 removing them entirely and making the ? icon in the header link to a relevant RTFM page in a new window (instead of opening RTFM in a modal)

**Comment by sottwell** [comment]
I use the close button frequently. And I would be more than happy if the Help button were easily configurable for each resource so I could link it to help pages for that resource, or its parent.

**Comment by exside** [comment]
Same here, at least for the help button, this would just be helpful if the global ? button in the menubar could be contextualized (as the "help" button should) and configured depending on where the user currently is...this is right now just possible with a very hacky .htacces thing (I need to have a completely custom "internal" page loaded into the modal for my custom help system)...

Close doesn't really disturb me, but for some users it may be helpful to have  "a way out", no?

**Comment by sottwell** [comment]
Actually simply clicking on the Dashboard icon has the same effect as the Close button, so I suppose it's really redundant.

**Comment by exside** [comment]
yes, but doesn the avarage editor user know that? or do we scare them =)...would be an interesting scenario for a hotjar type of analysis =), do people actually use that button...

**Comment by sottwell** [comment]
My clients know, if I feel it worth while to create such documentation. If they don't tell their editors, that's their problem.
